apps/examples/bundle_test/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_BUNDLE_TEST
+	bool "Bundle Test Example"
+	default n
```

Comment:
I think it should have a dependency on CONFIG_TIZEN_BUNDLE

Reply:
Thanks for the review. I'll add it.

Reply:
I'm not sure it should have dependency on  CONFIG_TIZEN_BUNDLE.
It is not only for TIZEN.